### PR TITLE
KIN-77 Whatsapp button public view front end

### PR DESCRIPTION
**Pull request by CarlosMay7**
[KIN-77](https://loopcrack.atlassian.net/browse/KIN-77) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

The clients need a fast way to contact the company in order to get more information about some products

### Solution

Create a whatsapp button that redirects the client to their whatsaap aplication and open the company chat with a default message

### Requirements checklist

Include a list of requirements necessary to consider this pull request complete

- [x] The button must be the WhatsApp logo

- [x] The button will always be fixed in one position

- [x] The button will redirect opening a new tab, not replacing the current one.

### Testing

Describe any tests (manual or automated) executed to verify your changes

- [x] Manually tested the button

## Demo

image
image

[KIN-77]: https://loopcrack.atlassian.net/browse/KIN-77?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/Views/public/templates/whatsapp.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php $this->section('js'); ?>
+<script src="assets/js/public/whatsapp.js"></script>
+<?php $this->endSection(); ?>
+<button class="btn-whatsapp" id="btn-whatsapp">
```
Comment (LUGAMAFE) [inline app/Views/public/templates/whatsapp.php:4]: @CarlosMay7 creo que seria mas facil si esto fuera un anchor con un href a la direccion que pusiste en el javascript no?

algo asi 

```suggestion
<a target="_blank" href="https://api.whatsapp.com/send?phone=.....">
```

**Review by LUGAMAFE — APPROVED** [review]

**Review by CarlosJCD — COMMENTED** [review]
Is it okay that the codeIgniter logo appears?.

Just asking because I genuinely don't know xd.

**Comment by CheloChidoone** [comment]
> Is it okay that the codeIgniter logo appears?.
> 
> Just asking because I genuinely don't know xd.

@CarlosJCD ese logo de codeigniter que aparece es el debug toolbar, si das click en el veras que sale una barra con unas funcionalidades para debug, no hay problema de que aparezca, ese toolbar aparece porque tenemos el `CI_ENVIRONMENT` como `development` en el `.env`, si lo cambiamos a `production` ya no aparece, asi que no hay problema con eso.

**Review by CheloChidoone — APPROVED** [review]

**Review by Oswaldo-Chan — APPROVED** [review]

File: app/Views/public/templates/whatsapp.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<a target="_blank" href="https://api.whatsapp.com/send/?phone=529993102600&text=*_WHATSAPP KINUB_*%0A*Hola, solicito información sobre dispositivos*">
```
Comment (Mauriciocr207) [inline app/Views/public/templates/whatsapp.php:1]: It´s not a good practice use button elements inside anchor elements in html.
It´s better use
`<a class="btn-whatsapp"><svg>...</svg></a>`
[reference](https://www.freecodecamp.org/espanol/news/ejemplos-de-codigo-de-enlaces-de-botones-html-como-crear-hipervinculos-html-usando-el-atributo-href-en-las-etiquetas/#:~:text=Los%20botones%20se%20suponen%20que,de%20una%20etiqueta%20de%20anclaje.)

**Review by Mauriciocr207 — CHANGES_REQUESTED** [review]

File: src/sass/public/UI/whatsapp/_whatsapp.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.btn-whatsapp {
+    position: fixed;
```
Comment (Mauriciocr207) [inline src/sass/public/UI/whatsapp/_whatsapp.scss:2]: sad

**Review by FernandoJoachin — APPROVED** [review]

**Review by CarlosJCD — APPROVED** [review]

**Review by Mauriciocr207 — APPROVED** [review]

**Pull request merged**